src/freedv_interface.cpp

```diff
@@ -59,6 +59,24 @@ void FreeDVInterface::OnReliableTextRx_(reliable_text_t rt, const char* txt_ptr,
     reliable_text_reset(rt);
 }
 
+float FreeDVInterface::GetMinimumSNR_(int mode)
+{
+    switch(mode)
+    {
+        case FREEDV_MODE_700C:
+            return 2.0;
+        case FREEDV_MODE_700D:
+            return -2.0f;
+        case FREEDV_MODE_700E:
+            return 1.0f;
+        case FREEDV_MODE_1600:
+            return 4.0f;
+        case FREEDV_MODE_2020:
+            return 2.0f;
+        default:
+            return 0.0f;
+    }
+}
 void FreeDVInterface::start(int txMode, int fifoSizeMs, bool singleRxThread, bool usingReliableText)
```

Comment:
This table could be a `freedv_api.h` getter type function, ie a property returned by the FreeDV API.  But that would mean a `codec2` PR so not really necessary right now :slightly_smiling_face: